wasm/src/account/account.rs

```diff
@@ -47,6 +47,18 @@ impl Account {
         }
     }
 
+    #[wasm_bindgen]
+    pub fn from_seed(seed: &[u8]) -> Self {
```

Comment:
This method can be shortened. In addition, you should use `ChaChaRng`.
```rust
AccountNative::new(&mut ChaChaRng::seed_from_u64(seed))
```
Please see our implementation in the CLI of this repository for an example.